dashboard/app/helpers/application_helper.rb

```diff
@@ -30,11 +30,6 @@ def ago(from_time)
     "#{s} ago"
   end
 
-  def format_xml(xml)
-    doc = Nokogiri::XML(xml)
-    doc.to_xhtml
-  end
```

Comment:
Note that this method is used in dashboard/app/views/shared/_solution.html.haml, itself being deleted in this PR.